### ISSUE TITLE
Issue parsing embedded strings

### ISSUE THREAD

**Issue by murpheywa** [adhocore/php-json-comment#15]
Given this valid JSON
```json
{
	"jo": "{\"url\":\"http://foo.bar\"}"
}
```

Comment::strip() returns

```json
{
        "js": "{\"url\":\"http:
}
```

**Comment by adhocore** [comment]
issue just confirmed, thanks for reporting. will be fixed in a few days.

**Comment by adhocore** [comment]
fixed in #16 n released as [v1.2.0](https://github.com/adhocore/php-json-comment/releases/tag/1.2.0)
